Doc/whatsnew/3.12.rst

```diff
@@ -1739,7 +1739,7 @@ unittest
 
   * Undocumented :meth:`TestLoader.loadTestsFromModule
     <unittest.TestLoader.loadTestsFromModule>` parameter *use_load_tests*
-    (deprecated and ignored since Python 3.2).
+    (deprecated and ignored since Python 3.5).
```

Comment:
Nice catch thank you.
Could you put this in a `.. deprecated:: Python3.5` directive ?

Reply:
Thanks for taking a look. I only see uses of that directive in the actual documentation relating to the method itself, not here in the changelog. I'm reluctant to add the first such directive in this /whatsnew directory.

Reply:
Fair enough. I guess this may be impacted by the bigger topic mentioned in https://github.com/python/cpython/pull/109843#issuecomment-2122904380 having to do with moving deprecation and removal items out of news.